keyboards/pom_keyboards/tnln95/tnln95.h

```diff
@@ -21,6 +21,7 @@
 // The first section contains all of the arguments
 // The second converts the arguments into a two-dimensional array
 
+/* commented this out to prevent confusion in the configurator
```

Comment:
Don't do this, instead add `LAYOUT_all` to the info.json.

Reply:
the full layout has many overlap keys so it might be hard to use.
but I will try to arrange the keys.

Reply:
I added `LAYOUT_all` to info.json